src/common-utils/main.sh

```diff
@@ -543,7 +542,7 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
 
         # Add devcontainer .zshrc template
         if [ "$INSTALL_OH_MY_ZSH_CONFIG" = "true" ]; then
-            echo -e "$(cat "${template_path}")\nDISABLE_AUTO_UPDATE=true\nDISABLE_UPDATE_PROMPT=true" > ${user_rc_file}
+            echo -e "$(cat "${template_path}")\n$(cat ${user_rc_file})\nDISABLE_AUTO_UPDATE=true\nDISABLE_UPDATE_PROMPT=true" > ${user_rc_file}
```

Comment:
Won't this throw file not found error?

Reply:
if statement again introduced to check user_rc_file exist.
if yes, appending the file content to template_path file content and overwritting the user_rc_file
if no, just using the template_path content and overwritting the user_rc_file which creates the file too